agenta-web/src/components/pages/overview/abTestingEvaluation/AbTestingEvalOverview.tsx

```diff
@@ -0,0 +1,440 @@
+        },
+        "& .ant-statistic-content-suffix": {
+            fontSize: 16,
+            color: theme.colorError,
+        },
+    },
+    stat: {
+        "& .ant-statistic-content-value": {
+            fontSize: 16,
+            color: theme.colorPrimary,
+        },
+        "& .ant-statistic-content-suffix": {
+            fontSize: 16,
+            color: theme.colorPrimary,
+        },
+    },
+    statGood: {
+        "& .ant-statistic-content-value": {
+            fontSize: 16,
+            color: theme.colorSuccess,
+        },
+        "& .ant-statistic-content-suffix": {
+            fontSize: 16,
+            color: theme.colorSuccess,
+        },
+    },
+}))
+
+const AbTestingEvalOverview = () => {
+    const classes = useStyles()
+    const router = useRouter()
+    const appId = router.query.app_id as string
+
+    const [evaluationsList, setEvaluationsList] = useState<HumanEvaluationListTableDataType[]>([])
+    const [fetchingEvaluations, setFetchingEvaluations] = useState(false)
+    const [isEvalModalOpen, setIsEvalModalOpen] = useState(false)
+    const [selectedEvalRecord, setSelectedEvalRecord] = useState<HumanEvaluationListTableDataType>()
+    const [isDeleteEvalModalOpen, setIsDeleteEvalModalOpen] = useState(false)
+
+    useEffect(() => {
+        if (!appId) return
+
+        const fetchEvaluations = async () => {
+            try {
+                setFetchingEvaluations(true)
+                const evals = await fetchAllLoadEvaluations(appId)
+
+                const fetchPromises = evals.map(async (item: any) => {
+                    return fetchEvaluationResults(item.id)
+                        .then((results) => {
+                            if (item.evaluation_type === EvaluationType.human_a_b_testing) {
+                                if (Object.keys(results.votes_data).length > 0) {
+                                    return abTestingEvaluationTransformer({item, results})
+                                }
+                            }
+                        })
+                        .catch((err) => console.error(err))
+                })
+
+                const results = (await Promise.all(fetchPromises))
+                    .filter((evaluation) => evaluation !== undefined)
+                    .sort(
+                        (a, b) =>
+                            new Date(b.createdAt || 0).getTime() -
+                            new Date(a.createdAt || 0).getTime(),
+                    )
+                    .slice(0, 5)
+
+                setEvaluationsList(results)
+            } catch (error) {
+                console.error(error)
+            } finally {
+                setFetchingEvaluations(false)
+            }
+        }
+
+        fetchEvaluations()
+    }, [appId])
+
+    const handleNavigation = (variantName: string, revisionNum: string) => {
+        router.push(`/apps/${appId}/playground?variant=${variantName}&revision=${revisionNum}`)
```

Comment:
I have noticed that this refreshes the whole page. Shouldn't the right behavior is to only update the right part of the page?